docs/examples/searchguide/expandableExample.tsx

```diff
@@ -0,0 +1,40 @@
+import { Fragment, useRef, useState } from 'react';
+import { Box, Flex, Popover, SearchGuide } from 'gestalt';
+
+export default function Example() {
+  const [showOutfits, setShowOutfits] = useState(false);
+  const anchorRef = useRef<HTMLElement | null>(null);
+
+  return (
+    <Fragment>
+      <Flex alignContent="stretch" alignItems="center" gap={2} justifyContent="center" width="100%">
+        <SearchGuide color="01" text="Designs" />
+        <SearchGuide
+          // @ts-expect-error - TS2322 - Type 'MutableRefObject<HTMLElement | null>' is not assignable to type 'LegacyRef<HTMLButtonElement> | undefined'.
+          ref={anchorRef}
+          color="02"
+          expandable
+          onClick={() => setShowOutfits((showing) => !showing)}
+          text="Outfits"
+        />
+        <SearchGuide color="03" text="Vintage" />
+      </Flex>
+      {showOutfits && (
+        <Popover
+          anchor={anchorRef.current}
+          idealDirection="down"
+          onDismiss={() => setShowOutfits(false)}
+          size="flexible"
+        >
+          <Box height={120} overflow="scrollX" padding={4}>
+            <Flex direction="row" gap={2} wrap>
+              <SearchGuide color="02" text="Casual" />
+              <SearchGuide color="02" text="Formal" />
```

Comment:
1. use just one where you implement the prop, not 3
2. change the colors in children
3. try layer and check if it stays accesible. it should add the focus to the first SearchGuide on keyboard navigation.

That would be a good example. if it breaks let me knwo

Reply:
Also, apply all the accessibilty props!


Reuse this example in the accessibility section.

Reply:
Can we do 1 full implementation in one example, and then another example  where we can see all this permutations? 
![Screenshot by Dropbox Capture](https://github.com/user-attachments/assets/d41fe7c4-cf1a-4f4f-bc61-91b21235d454)

Reply:
![Screenshot by Dropbox Capture](https://github.com/user-attachments/assets/7aa9381a-daec-436e-81e6-ce73146eeb17)
 shouldn't this one become selected?